src/Symfony/Component/Stopwatch/StopwatchEvent.php

```diff
@@ -177,7 +177,7 @@ public function getDuration()
 
         for ($i = 0; $i < $left; ++$i) {
             $index = $stopped + $i;
-            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow());
+            $periods[] = new StopwatchPeriod($this->started[$index], $this->getNow(), true);
```

Comment:
@keradus Using `func_get_args/func_num_args()` is the only way to have proper upgrade paths when adding an argument to a method. 
They can make the code less readable for a period, but that's for making it better at the end, that is what BC layers are all about.

See e.g.
https://github.com/symfony/symfony/blob/3.3/src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php#L61
https://github.com/symfony/symfony/blob/3.3/src/Symfony/Component/HttpFoundation/Request.php#L597
https://github.com/symfony/symfony/blob/3.3/src/Symfony/Component/DependencyInjection/Definition.php#L805
Worst: https://github.com/symfony/symfony/blob/af4703f6f209a9abc387f81b98e00fad46b89a38/src/Symfony/Bundle/SecurityBundle/Security/FirewallMap.php#L25..#L106

All of these have been removed from master already, and I guess we're all happy about what the changes involving those layers will provide.